src/main/kotlin/no/nav/familie/ef/sak/api/dto/BehandlingHistorikkDto.kt

```diff
@@ -0,0 +1,11 @@
+package no.nav.familie.ef.sak.api.dto
+
+import no.nav.familie.ef.sak.service.steg.StegType
+import java.time.LocalDateTime
+import java.util.*
+
+data class BehandlingshistorikkDto(val behandlingId: UUID,
```

Comment:
Litt usikker på denne.
I andre entiteter bruker vi Sporbar, den får med seg opprettetAv automatisk. @charliemidtlyng tanker? 
Trenger vi både navn og mail?

Hvis vi ikke trenger hele den Sporbar, så tenker jeg kanskje att dette burde være tilstrekkelig. Samtidig er det kanskje fint å bruke.
```
data class Behandlingshistorikk(@Id
                                val id: UUID = UUID.randomUUID(),
                                val behandlingId: UUID,
                                val steg: StegType,
                                val opprettetAv: String,
                                val opprettetTid: LocalDateTime = LocalDateTime.now())
```

Reply:
Denne er litt forskjellig fra Sporbar: Vi ønsker navn på saksbehandler (visning), men det finnes ikke noen "endring" på et logginnslag og derfor er ikke det interessant her. 

Synes derfor at implementasjonen til @throndi fungerer godt.

Reply:
Ok, tenker nok att det kan være fint å bruke det samme ordbruket som i sporbar då. Slik att vi bruker `OpprettetAv`for selve mailen

```
data class Behandlingshistorikk(@Id
                                val id: UUID = UUID.randomUUID(),
                                val behandlingId: UUID,
                                val steg: StegType,
                                val opprettetAv: String = SikkerhetContext.hentSaksbehandler(),
                                val opprettetAvNavn: String = SikkerhetContext.hentSaksbehandlerNavn(),
                                val opprettetTid: LocalDateTime = LocalDateTime.now())
```

Reply:
Da er jeg enig 👍